### ISSUE TITLE
Migration from OTRS: Fix postgre base64 encoding of articles.

### ISSUE THREAD

**Issue by svenoe** [RotherOSS/otobo#800]
Postgresql base64 encodes some article data. We have to check whether this is handled correctly in the current version of the migration procedure. At some point it wasn't.

(A fix for the failed migration was to decode the encoded articles later - in Kernel/System/Ticket/Article/Backend/MIMEBase/ArticleStorageDB.pm:
```
#        if ( !$DBObject->GetDatabaseFunction('DirectBlob') && $Row[0] !~ m/ / ) {
if ( $Param{ArticleID} < 71944 ) {

            $Data = decode_base64( $Row[0] );
```

Note to self:
- [ ] 2020111010000047)

**Comment by bschmalhofer** [comment]
See also #601. Maybe #601 can be closed as a duplicate of #800.

**Comment by StefanRother-OTOBO** [comment]
Dublicate with Issue #1006, #800 and #601
